src/main/java/duke/command/MarkCommand.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;
+
+import duke.exception.KoraException;
+import duke.storage.Storage;
+import duke.task.Task;
+import duke.task.TaskList;
+
+public class MarkCommand extends Command {
+
+    private String commandMessage = "";
+    private int taskIndex;
+
+    public MarkCommand(String[] details) {
+        taskIndex = Integer.valueOf(details[0].replace("mark ", ""));
```

Comment:
what if command is not an Integer value? such as `mark a/ mark $`

Reply:
Thanks for pointing out. I will take note!